### PR TITLE
Add CI

### PR DESCRIPTION
**Pull request by TomAugspurger**

### CODE REVIEW

**Comment by TomAugspurger** [comment]
We may need to merge this before we can verify that it's actually working.

**Comment by TomAugspurger** [comment]
Oh it's running on my branch.

File: rechunker/algorithm.py [hunk 1]
```diff
@@ -3,7 +3,10 @@
 
 try:
     from math import prod
+
+    numpy_prod = False
```
Comment (rabernat) [inline rechunker/algorithm.py:7]: Apparently prod is not in math until python 3.8. Maybe we should just always use numpy here? Math has type annotations.

**Comment by TomAugspurger** [comment]
I think either is fine, I'm just trying to debug an issue on the CI. For
some reason Python is complaining about non-JSON serializable objects, I
think a scalar numpy.int64 value. Will figure it out.

**Comment by TomAugspurger** [comment]
All passing on my branch.

I'm not really sure why the conversion from NumPy to python ints was needed, but somewhere Zarr was trying to JSON serialize them (I think the metadata like shape / chunks).

**Pull request merged**